johnwayodi/sm-api
# #161406407 Move tests folder to project root directory

#### What does this PR do?
Relocate tests folder to the root directory of the project

#### Description of Task to be completed?
Ensure tests are still running successfully after relocation
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api.git`
2. Ensure you are in the cloned repo, **_sm-api_** by default.
3. `git checkout ch-tests-relocation-161406407`
4. Run the tests.

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161406407
#### Questions:
Any feedback on what to improve?
